.github/workflows/ci.yaml

```diff
@@ -0,0 +1,134 @@
+# Add a comment to force a change.
+
+name: Webhook CI
+
+on:
+  workflow_call:
+  push:
+    branches:
+      - release/v*
+    paths-ignore:
+      - '*.md'
+      - '.gitignore'
+      - 'CODEOWNERS'
+      - 'LICENSE'
+  pull_request:
+    paths-ignore:
+    - '*.md'
+    - '.gitignore'
+    - 'CODEOWNERS'
+    - 'LICENSE'
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    name: CI
+    runs-on : ubuntu-latest
+    strategy:
+      matrix:
+          arch:
+          - amd64
+          - arm64
+    steps:
+      - name : Checkout repository
+        uses : actions/checkout@v4
+        with :
+          fetch-depth : 0
+      - run : make ci
+        name: ci
+
+      - name : Save builds
+        run : |
+          mkdir -p dist/artifacts
+          cp bin/* dist/artifacts/
+      - name: Upload artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: webhook-build-artifacts-${{ matrix.arch }}
+          path: ./dist/artifacts
+
+  test:
+    name : integration-test
+    needs : [
+      build
+    ]
+    runs-on : ubuntu-latest
+    strategy:
+      matrix:
+        arch:
+        - amd64
+        - arm64
+    steps:
+      - name : Checkout repository
+        uses : actions/checkout@v4
+        with :
+          fetch-depth : 0
```

Comment:
Do we need to fetch the whole history from rancher/webhook? If not then I suggest sticking with the default.

Reply:
Yup, removed